src/utils/thread_helper.cpp

```diff
@@ -1382,6 +1385,26 @@ otError ThreadHelper::RetrieveTelemetryData(Mdns::Publisher &aPublisher, threadn
         // End of CoexMetrics section.
     }
 
+#if OTBR_ENABLE_LINK_METRICS_TELEMETRY
+    {
+        auto lowPowerMetrics = telemetryData.mutable_low_power_metrics();
+        // Begin of Link Metrics section.
+        for (const otNeighborInfo &neighborInfo : neighborTable)
+        {
+            otError             query_error;
+            otLinkMetricsValues values;
+
+            query_error = otLinkMetricsManagerGetMetricsValueByExtAddr(mInstance, &neighborInfo.mExtAddress, &values);
```

Comment:
Do you want to use VerifyOrExit here?

Reply:
Some neighbors doesn't support Link Metrics. So it is expected that some errors other than `OT_ERROR_NONE` happened.

Reply:
SG. Can you add a comment in the code about this?

Reply:
SG! Updated.